### ISSUE TITLE
After connecting to remote server theres no tree view

### ISSUE THREAD

**Issue by tmrouse** [icetee/remote-ftp#701]
To be able to see the tree view again you have to manually toggle the remote-ftp panel this is what its like after connecting. http://imgur.com/a/PYQ8g

**Comment by jpxd** [comment]
Does the `Show view on startup` setting in the package preferences help?

**Comment by tmrouse** [comment]
Having that option on/off does not make a difference :/

**Comment by jpxd** [comment]
Did you try: activate the option, completely close Atom and open the project again?

**Comment by tmrouse** [comment]
Yeah, still makes no difference also it does not start on startup unless i manually toggle it even with the option. Not sure if this is relavent but when i toggle it manually the first time on each reset of atom i get see error 433 not sure if that is to dow with this is did look but dint think it was.

**Comment by samifouad** [comment]
Do you have the latest version of this package and Atom?

Is there any other package that could be interfering with tree view visibility? To help troubleshoot what's going on you could temporarily disable other packages.

**Comment by tmrouse** [comment]
Im using the latest version of atom and remote-ftp. With every package disabled but remote-ftp it still does it.

**Comment by samifouad** [comment]
If you go to **File** then **Stylesheet** is there any special styling for **.tree-view** or **.remote-ftp-view**?

If you add this:

```
.tree-view {
  display: inline-block;
}
```

and save the file, does anything happen?

**Comment by hatzopoulos** [comment]
~~Identical~~ Similar report as https://github.com/mgrenier/remote-ftp/issues/643

**Comment by tmrouse** [comment]
Hello, adding: 

```
`.tree-view {
  display: inline-block;
}
```

Dose nothing.`

**Comment by shulman1001** [comment]
hey, i have to this problem.
i see when i connect to another server all is fine.
why this happen? and what i can to do please?
